src/generic/datavgen.cpp

```diff
@@ -820,7 +821,7 @@ class wxDataViewMainWindow: public wxWindow
 
     int GetLineStart( unsigned int row ) const;  // row * m_lineHeight in fixed mode
     int GetLineHeight( unsigned int row ) const; // m_lineHeight in fixed mode
-    int GetLineAt( unsigned int y ) const;       // y / m_lineHeight in fixed mode
+    int GetLineAt( int y ) const;                // y / m_lineHeight in fixed mode
```

Comment:
I know you mentioned this in a PR comment, but I still don't really understand why did this need to be changed. Is `GetLineAt()` supposed to be called with non-positive values of `y`? IMO this shouldn't be the case, i.e. we need to find out where does this happen and fix the caller. And if it doesn't happen, then this change should be just reverted because it doesn't change anything, of course.

FWIW I wouldn't be against using `int` -- but then we need to do it everywhere, not just in a single function here. And to avoid changing too many things at once, for now I'd rather avoid changing this.

Reply:
allright i changed this back.